packages/manager/apps/telecom/src/app/telecom/telephony/service/contact/contact.controller.js

```diff
@@ -375,7 +379,7 @@ export default /* @ngInject */ function TelecomTelephonyServiceContactCtrl(
   // arrondissements pour paris 75xxx, marseille 13xxx et lyon 69xxx (92 izi?)
```

Comment:
Maybe following comment can be updated?

/cc @Jisay @blaryjp

Reply:
commit has been amend to update comment